python/ray/train/tests/_lightning_utils.py

```diff
@@ -0,0 +1,79 @@
+import pytorch_lightning as pl
```

Comment:
why prefix the file with ``_``? just curious.

Reply:
Means not for tests? I just saw another `_hugginface_data.py` file and followed its naming format.